src/_data/side-nav.yml

```diff
@@ -6,6 +6,8 @@
       permalink: /samples
     - title: Codelabs
       permalink: /codelabs
+    - title: Install Dart
+      permalink: /install
 
```

Comment:
Thanks for doing this!

I'm thinking this should be under TOOLS instead. (A 3-item QUICK START is already long.)